db/mongoose.js

```diff
@@ -332,14 +333,20 @@ const twapCreation = async (req, res, next) => {
       time = time * 1000;
 
       if (assetPairArray[assetPairAddress].value == "0xedf187890af846bd59f560827ebd2091c49b75df") {
+        collateral = "USDC";
```

Comment:
this value should be dynamic getting it from the assets.json collateral based of the asset